libres/testjenkins.sh

```diff
@@ -143,7 +143,7 @@ run_pytest_equinor () {
 
 	# Avoid implicitly loaded cwd modules
 	pushd ${LIBRES_BUILD}
-	python -m pytest -m "equinor_test" --durations=10 ${ERT_SOURCE_ROOT}/libres/tests
+	python -m pytest -m "equinor_test" --durations=10 ${ERT_SOURCE_ROOT}/tests/libres_tests
```

Comment:
Should we maybe move this file to the other Jenkins CI files ( `ert/ci/jenkins` ) ? And rename to `testlibres.sh` or something?

Reply:
this will be picked up in #1809